src/smallestPrimeFactor.js

```diff
@@ -13,11 +13,32 @@
  */
 function smallestPrimeFactor(num) {
   // This is your job. :)
+  let primeFactor = 1
+  for (let i = num - 1; i > 1; i--) {
```

Comment:
Iterate up from `2` and return as soon as you find the first factor (which must be prime).